src/components/TruncatedList/TruncatedList.stories.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @file Truncated list stories.
+ * @copyright IBM Security 2020
+ */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { number, select, color, text } from '@storybook/addon-knobs';
+import { components } from '../../../.storybook';
+import TruncatedList from '.';
+import { createChildrenArray } from './_mocks_';
+
+storiesOf(components(TruncatedList.name), module).add(
+  'default',
+  () => (
+    <TruncatedList
```

Comment:
Could you wrap this in a demo container with a set width to avoid the jumpiness in the demo?